install/gcp/installer/configs/config.py

```diff
@@ -33,6 +33,12 @@ def __init__(self, **kwargs):
                               datetime.datetime.now().strftime('%Y%m%d%H%M%S'))
         self.identifier = None
         self.force_no_cloudshell = bool(kwargs.get('no_cloudshell'))
+        self.project_id = kwargs.get('project_id') or None
```

Comment:
.get will return None if there is no project_id key, so the or None part is redundant

Reply:
Thanks. Change made